subcommand/server-acl-init/rules_test.go

```diff
@@ -493,31 +494,32 @@ namespace_prefix "prefix-" {
 	}
 }
 
+// There are three true/false settings so 8 permutations to test.
```

Comment:
Sorry in advance. I implemented these tests via a truth table that covers all permutations.